### PR TITLE
feat: scheduler configuration

### PR DESCRIPTION
**Pull request by hejkerooo**

### CODE REVIEW

File: receive.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```
Comment (vikaskumr) [inline receive.sh:2]: are these for testing on server?

Comment (hejkerooo) [reply receive.sh:2]: forgot to remove them - thanks for pointing out, I was ddosing myself to test performance

File: apps/dsb-client-gateway-scheduler/src/app/modules/heartbeat/service/heartbeat.service.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { Cron, SchedulerRegistry } from '@nestjs/schedule';
+import { CronJob } from 'cron';
+
+@Injectable()
+export class HeartbeatService {
+  protected readonly logger = new Logger(HeartbeatService.name);
+
+  constructor(protected readonly schedulerRegistry: SchedulerRegistry) {}
+
+  @Cron('30 * * * * *', {
```
Comment (vikaskumr) [inline apps/dsb-client-gateway-scheduler/src/app/modules/heartbeat/service/heartbeat.service.ts:11]: you have maintained all the other times in.env file then why not this one?

**Review by vikaskumr — CHANGES_REQUESTED** [review]
LGTM only some changes

**Pull request merged**